Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.yml

```diff
@@ -193,7 +193,7 @@ script:
     - contextPath: MITREATTACK.value
       description: MITRE ATTACK Attack Pattern value.
       type: String
-  dockerimage: demisto/taxii2:1.0.0.21842
+  dockerimage: demisto/taxii2:1.0.0.23423
```

Comment:
Please don't update the image, the taxi image broke some functionality in this integration